src/screens/transaction_details/components/msg/unknown/index.tsx

```diff
@@ -11,7 +11,7 @@ const Unknown = (props: {
   return (
     <pre className={classes.root}>
       <code>
-        {JSON.stringify(message.json, null, '\t')}
+        {JSON.stringify(message?.json || message, null, '\t')}
```

Comment:
Please revert back to 
```
{JSON.stringify(message.json, null, '\t')}
```

As explained before, turning the model in to json is still different form displaying the real raw message

Reply:
Updated!